### ISSUE TITLE
Safe Html for angular

### ISSUE THREAD

**Issue by tranbaquan** [ckeditor/ckeditor5-angular#282]
I added custom editor with 'sourceEditing', 'htmlEmbed', 'fontColor', ... toolbars. But when bidding model it's not work cause angular security check.
example: 
I insert an youtube iframe and binding using ngModel. But data bidding always skip iframe tag.

**Comment by FilipTokarski** [comment]
Hi, could you describe the issue more precisely and add detailed steps to reproduce this? It would be also good to know your editor configuration, plugins installed, etc.

**Comment by tranbaquan** [comment]
Hi @FilipTokarski, 
I create custom build package from https://ckeditor.com/ckeditor-5/online-builder/ and install to my project. 
Then I got the issue as my described above. After times trying to config them, I deleted the custom build and create another one. Howerver, this time it's work fine.
Maybe I missed something when choosing feature but I'm sure both 2 times I already include htmlEmbed and sourceEditing and I don't know what feature missed in the first time.
